src/library/process.js

```diff
@@ -59,6 +70,12 @@ export const processMessage = async (AWS, region, service, account,
     }
   }
 
+  // CR: Mickey: I would prefer not to arbitrarily abbreviate this variable name
+  //    especially when processResponse is later abbreviated differently later in this file.
+  //    Wait is this actually `processResult`?  I think that would be the most clear
+  //    since the full object is not a response from the handler and also has status details
+  //    For files that are /behind the scenes/ I don't feel that the loss of clarity is worth
+  //    saving a few keystrokes.
```

Comment:
@MickeyMcc There are instances where people have hated me for extremely long variable names. So, I was trying to keep such people happy. However, I actually like the variable names to completely explain what they are for.